src/net/sourceforge/kolmafia/session/QuestManager.java

```diff
@@ -64,6 +62,25 @@ public class QuestManager {
   private static final Pattern DJ_MEAT_PATTERN = Pattern.compile("collect (.*?) Meat for the DJ");
   private static final Pattern TRASH_PATTERN = Pattern.compile("you clean up (\\d+) ");
 
+  private static List<String> friarElbowNCs =
+      List.of(
+          "Deep Imp Act",
+          "Imp Art, Some Wisdom",
+          "A Secret, But Not the Secret You're Looking For",
+          "Butter Knife?  I'll Take the Knife");
+  private static List<String> friarNeckNCs =
+      List.of(
+          "How Do We Do It? Quaint and Curious Volume!",
+          "Strike One!",
+          "Olive My Love To You, Oh.",
+          "Dodecahedrariffic!");
+  private static List<String> friarHeartNCs =
+      List.of(
```

Comment:
One last thing - since we're very close and you have it loaded now I think these should be Sets not Lists if there's nothing major stopping you doing so